use_case_examples/cifar/cifar_brevitas_training/evaluate_torch_cml.py

```diff
@@ -123,6 +123,7 @@ def main(args):
                 if rounding_threshold_bits is not None
                 else None
             ),
+            device=COMPILATION_DEVICE,
```

Comment:
this isn't defined anywhere

Reply:
I corrected it, when I tested on my machine